test/WebJobs.Script.Tests/WebJobs.Script.Tests.csproj

```diff
@@ -558,7 +558,7 @@
     <None Include="TestScripts\Bash\TimeoutSync\run.sh">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
-    <None Include="TestScripts\Content\Node\function.json">
+    <None Include="TestScripts\Node\HttpTriggerContentType\function.json">
```

Comment:
Why do we need a separate function script for this at all? It seems like it can just be encoded into the existing HttpTrigger-Scenarios function?